RelayDotNet/Relay.cs

```diff
@@ -809,6 +809,36 @@ private async Task<RunningRelayWorkflow> GetRunningRelayWorkflowOrThrow(IRelayWo
 
             return runningRelayWorkflow;
         }
+
+        /// <summary>
+        /// Get the source URN from a workflow trigger
+        /// </summary>
+        /// <param name="trigger">workflow trigger</param>
+        /// <returns>the source URN as a string from the trigger</returns>
+        public static string GetSourceUriFromTrigger(IDictionary<string, object> trigger) 
```

Comment:
Agree that this looks like a duplicate of GetSourceUriFromStartEvent. Perhaps just the method name needs to be consistent.

Reply:
There are all kinds of possibilities here. My initial preference at this point is that GetSourceUriFromStartEvent is a more clear description than getSourceUriFromTrigger, as the methods are passed the start event. And the methods should be named as closely as makes-sense-and-reasonably-possible across all the SDKs. Meaning Java should also probably be getSourceUriFromStartEvent.

Reply:
I agree - will update now